modules/cluster-logging-rn-5.5.7.adoc

```diff
@@ -0,0 +1,22 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-7_{context}"]
+= Logging 5.5.7
+This release includes link:https://access.redhat.com/errata/RHSA-2023:0633[OpenShift Logging Bug Fix Release 5.5.7].
+
+[id="openshift-logging-5-5-7-bug-fixes"]
+== Bug fixes
+* Before this update, the LokiStack Gateway Labels Enforcer generated parsing errors for valid LogQL queries when using combined label filters with boolean expressions. With this update, the LokiStack LogQL implementation supports label filters with boolean expression and resolves the issue. (link:https://issues.redhat.com/browse/LOG-3534[LOG-3534]), (link:https://issues.redhat.com/browse/LOG-3441[LOG-3441])
```

Comment:
LOG-3441 is for 5.6.z release. Not sure if we need to mention it under 5.5.7?

Reply:
And again, something slipped my view. I think LOG-3441 is just the original issue that 3534 was cloned from and can be removed.

Reply:
Removed, thanks team.